k8s/overlays/nerc-ocp-infra/patches/bc-xdmod-dev.yaml

```diff
@@ -6,4 +6,4 @@ spec:
   source:
     type: Git
     git:
-      ref: IncludeNamespace
+      ref: FixInvalidBc
```

Comment:
there should be a newline at the end of file.

Reply:
ok